src/main/java/org/kizombadev/markdownparser/Program.java

```diff
@@ -23,4 +23,14 @@
 
 
 public class Program {
+
+    public static void main(String[] args) {
+
+        try {
+            new Application().execute(args);
+
+        } catch (Exception e) {
+            System.out.println(e);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)